tests/test_interpolatedimage.py

```diff
@@ -880,6 +880,36 @@ def test_conserve_dc():
     t2 = time.time()
     print 'time for %s = %.2f'%(funcname(),t2-t1)
 
+def test_stepk_maxk():
+    """Test options to specify (or not) stepk and maxk.
+    """
+    import time
+    t1 = time.time()
+    import numpy
+
+    scale = 0.18
+    n = 101 # use an odd number so profile doesn't get recentered at all, modifying stepk
+
+    obj = galsim.Exponential(half_light_radius=2.*scale)
+    im = galsim.Image(n, n)
+    im = obj.drawImage(image=im, scale=scale)
+    int_im = galsim.InterpolatedImage(im)
+
+    step_k_val = int_im.stepK()
+    max_k_val = int_im.maxK()
+
+    mult_val = 0.9
+    new_int_im = galsim.InterpolatedImage(im, _force_stepk=mult_val*step_k_val,
+                                          _force_maxk=mult_val*max_k_val)
+    numpy.testing.assert_almost_equal(
+        new_int_im.stepK(), mult_val*step_k_val, decimal=7,
+        err_msg='InterpolatedImage did not adopt forced minimum value for stepK')
```

Comment:
Is this really a *minimum* value for stepK and not just *the* value?

and similarly maxK?